### PR TITLE
Milestone 8: Add accessibility 🕶

### PR DESCRIPTION
**Pull request by mdg0410**
## Accessibility Report

- ## **Page titles**

The page title is poor and was changed to a more descriptive page title.

https://github.com/mdg0410/Portfolio/blob/3ab6b2af712ea51580c927c15cbce22cf08aefff/index.html#L7

- ## **Image text alternatives**

Add alternative texts to images that are important for understanding the content and add null alternative texts to decorative images.

> Alternative text important

https://github.com/mdg0410/Portfolio/blob/f9d1bc8f9d35b8798144ac825429c9b47799335a/index.html#L41
https://github.com/mdg0410/Portfolio/blob/f9d1bc8f9d35b8798144ac825429c9b47799335a/index.html#L76
https://github.com/mdg0410/Portfolio/blob/f9d1bc8f9d35b8798144ac825429c9b47799335a/index.html#L77
https://github.com/mdg0410/Portfolio/blob/f9d1bc8f9d35b8798144ac825429c9b47799335a/index.html#L78
https://github.com/mdg0410/Portfolio/blob/f9d1bc8f9d35b8798144ac825429c9b47799335a/index.html#L79
https://github.com/mdg0410/Portfolio/blob/f9d1bc8f9d35b8798144ac825429c9b47799335a/index.html#L80
https://github.com/mdg0410/Portfolio/blob/f9d1bc8f9d35b8798144ac825429c9b47799335a/index.html#L96
https://github.com/mdg0410/Portfolio/blob/f9d1bc8f9d35b8798144ac825429c9b47799335a/index.html#L131
https://github.com/mdg0410/Portfolio/blob/f9d1bc8f9d35b8798144ac825429c9b47799335a/index.html#L168
https://github.com/mdg0410/Portfolio/blob/f9d1bc8f9d35b8798144ac825429c9b47799335a/index.html#L205
https://github.com/mdg0410/Portfolio/blob/f9d1bc8f9d35b8798144ac825429c9b47799335a/index.html#L259
https://github.com/mdg0410/Portfolio/blob/f9d1bc8f9d35b8798144ac825429c9b47799335a/index.html#L260
https://github.com/mdg0410/Portfolio/blob/f9d1bc8f9d35b8798144ac825429c9b47799335a/index.html#L261
https://github.com/mdg0410/Portfolio/blob/f9d1bc8f9d35b8798144ac825429c9b47799335a/index.html#L262
https://github.com/mdg0410/Portfolio/blob/f9d1bc8f9d35b8798144ac825429c9b47799335a/index.html#L263
https://github.com/mdg0410/Portfolio/blob/f9d1bc8f9d35b8798144ac825429c9b47799335a/index.html#L279
https://github.com/mdg0410/Portfolio/blob/f9d1bc8f9d35b8798144ac825429c9b47799335a/index.html#L283
https://github.com/mdg0410/Portfolio/blob/f9d1bc8f9d35b8798144ac825429c9b47799335a/index.html#L287

> Null alternative text

https://github.com/mdg0410/Portfolio/blob/f9d1bc8f9d35b8798144ac825429c9b47799335a/index.html#L274
https://github.com/mdg0410/Portfolio/blob/f9d1bc8f9d35b8798144ac825429c9b47799335a/index.html#L297
https://github.com/mdg0410/Portfolio/blob/f9d1bc8f9d35b8798144ac825429c9b47799335a/index.html#L306

- ## **Text headings**

No issues found

- ## **Color contrast**

Change the color of the letters in the paragraph of the contact me section of the mobile version

https://github.com/mdg0410/Portfolio/blob/f9d1bc8f9d35b8798144ac825429c9b47799335a/styles.css#L1258

- ## **Resize**

No issues found

- ## **Interaction**

Add invisible `label` tags in the contact me section

https://github.com/mdg0410/Portfolio/blob/f9d1bc8f9d35b8798144ac825429c9b47799335a/index.html#L325
https://github.com/mdg0410/Portfolio/blob/f9d1bc8f9d35b8798144ac825429c9b47799335a/index.html#L328
https://github.com/mdg0410/Portfolio/blob/f9d1bc8f9d35b8798144ac825429c9b47799335a/index.html#L330

- ## **Moving content**

Change the focus when placed in an input tag

> Version mobile

https://github.com/mdg0410/Portfolio/blob/f9d1bc8f9d35b8798144ac825429c9b47799335a/styles.css#L624
https://github.com/mdg0410/Portfolio/blob/f9d1bc8f9d35b8798144ac825429c9b47799335a/styles.css#L625

> Version desktop

https://github.com/mdg0410/Portfolio/blob/f9d1bc8f9d35b8798144ac825429c9b47799335a/styles.css#L1269
https://github.com/mdg0410/Portfolio/blob/f9d1bc8f9d35b8798144ac825429c9b47799335a/styles.css#L1270

- ## **Multimedia**

No issues found

- ## **The basic structure of the page**

No issues found

### CODE REVIEW

**Review by Qoosim — COMMENTED** [review]
#  *Status: Invalid Request* :red_circle: 

Hi @team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 

- [ ] missed `Key Features` heading in your README fiile. Kindly fix that.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by mdg0410** [comment]
> Correction 

- Add `Key features` in README file
- Update collaborator `@baezluise` in README file

File: styles.css [hunk 1]
```diff
@@ -42,7 +42,7 @@
   }
 
   .my-logo {
-    font-family: 'Poppins', sans-serif;
+    font-family: "Poppins", sans-serif;
```
Comment (osamaashraf6) [inline styles.css:45]: - [ ] I see 😃that there is a problem with the contrast color here at the logo, so kindly try to fix this contrast where The page has a minimum contrast ratio of at least 4.5:1 for normal-size text.😊

| The logo |
| --- |
|   image  |

File: index.html [hunk 2]
```diff
@@ -271,20 +271,20 @@ <h2 class="primary-text-about-me">About
...
                   <div class="frame-list">
-                    <img class="frame-img" src="./img/Ellipse 1.svg">
+                    <img class="frame-img" src="./img/Ellipse 1.svg" alt="JavaScript skill">
                     <span class="frame-text">javaScript</span>
                   </div>
                   <div class="frame-list">
-                    <img class="frame-img" src="./img/Ellipse 1-1.svg">
+                    <img class="frame-img" src="./img/Ellipse 1-1.svg" alt="HTML skill">
                     <span class="frame-text">HTML</span>
                   </div>
```
Comment (osamaashraf6) [inline index.html:285]: - [ ]  I see 😃that there is a problem with the contrast color here at the li, so kindly try to fix this contrast of all li of languages where The page has a minimum contrast ratio of at least 4.5:1 for normal-size text.😊

| The li|
| --- |
|   image |

File: styles.css [hunk 3]
```diff
@@ -169,14 +169,18 @@
   }
 
   .lets-connect {
-    font-family: 'Poppins', sans-serif;
+    font-family: "Poppins", sans-serif;
```
Comment (osamaashraf6) [inline styles.css:172]: - [ ]  I see 😃that there is a problem with the contrast color here at the text ```let's connect```, so kindly try to fix this contrast of the text like ```let's connect``` where The page has a minimum contrast ratio of at least 4.5:1 for normal-size text.😊

| Look Here |
| --- |
|   image |

File: index.html [hunk 4]
```diff
@@ -322,10 +322,13 @@ <h2 class="primary-text-about-me">About
...
+        <input class="input" name="user_name" id="name" type="name" placeholder="Enter your name" pattern="[A-Za-z/ ]{0,30}"
           required>
-        <input class="input" name="Email" id="email" type="email" placeholder="Enter your email" required>
-        <textarea class="input" name="message" id="message" placeholder="Write your message here" maxlength="500" required></textarea>
+          <label for="email" id="email"></label>
+        <input class="input" name="user_email" id="email" type="email" placeholder="Enter your email" required>
+        <label for="message"></label>
+        <textarea class="input" name="user_message" id="message" placeholder="Write your message here" maxlength="500" required></textarea>
         <button class="submit-button" id="form-submit" type="submit">Get in touch</button>
       </form>
```
Comment (osamaashraf6) [inline index.html:333]: - [ ]  I see 😃that there is a problem with the contrast color here at the submit button, so kindly try to fix this contrast of submit button where The page has a minimum contrast ratio of at least 4.5:1 for normal-size text.😊

| Look Here|
| --- |
|   image |

**Review by osamaashraf6 — CHANGES_REQUESTED** [review]
Hi @mdg0410 👋, 

Good job so far! 👏🥳
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights ✔️
- [x] Good title page
- [x] Descriptive READme file in a professional way.
- [x] No linter errors. 💯
- [x] Good usage in correcting GitHub flow
- [x] Nice design.




## Required Changes ♻️

- [ ] check the comments under the review.

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (mdg0410) [reply index.html:333]: ## Hi @osamaashraf6 🖐

I kindly ask you to help me with your opinion. 😊

I was playing with the contrasts and I found the solution that passes several tests but the color varies a little with the figma design.

> My question is:

Is there any problem if I use this contrast?

imagen

File: styles.css (hunk 1 above)
Comment (mdg0410) [reply styles.css:45]: > Correction

|My Logo|Test contrast color|
|:---------:|:-------------------:|
|imagen|imagen|

File: index.html (hunk 2 above)
Comment (mdg0410) [reply index.html:285]: > Correction

|Tags|Test contrast color|
|:---------:|:-------------------:|
|imagen|imagen|

File: styles.css (hunk 3 above)
Comment (mdg0410) [reply styles.css:172]: > Correction 

|Let's Connect|Test contrast color|
|:---------:|:-------------------:|
|imagen|imagen|

File: index.html (hunk 4 above)
Comment (mdg0410) [reply index.html:333]: > Correction

|Get in touch|Test contrast color|
|:---------:|:-------------------:|
|imagen|imagen|

**Review by AbayomiOlaoye — APPROVED** [review]
Hi @mdg0410 👋 😄 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

applause

## Highlights
- Great implementation of reviewer's suggestions :heavy_check_mark: 
- Good UI implementation :heavy_check_mark: 
- Linter checks are passing :heavy_check_mark: 
- Improved Accessibility score ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**